.gitignore

```diff
@@ -19,3 +19,7 @@ coverage
 
 # bundled files
 lib/bundle.js
+
+package-lock.json
```

Comment:
Surely we should be adding this. ["This file is intended to be committed into source repositories"](https://docs.npmjs.com/files/package-lock.json)